### PR TITLE
bugs resolution for change user password

### PR DESCRIPTION
**Pull request by daisyzarzo**
Fixes #

relates to https://github.com/bespoken/dashboard/issues/890
https://github.com/bespoken/dashboard/issues/893
https://github.com/bespoken/dashboard/issues/891
https://github.com/bespoken/dashboard/issues/892
https://github.com/bespoken/dashboard/issues/889
https://github.com/bespoken/dashboard/issues/895
https://github.com/bespoken/dashboard/issues/889
https://github.com/bespoken/dashboard/issues/894

### CODE REVIEW

**Comment by daisyzarzo** [comment]
image

**Comment by daisyzarzo** [comment]
image

**Comment by daisyzarzo** [comment]
image

File: src/components/MyAccount/Account.tsx [hunk 1]
```diff
@@ -125,14 +124,11 @@ export class Account extends React.Component<AccountProps, AccountState> {
...
     render() {
-        const userDetailPropertyList: SectionProperties[] = this.getUserDetailProperies();
-        const subscriptionPropertyList: SectionProperties[] = this.getSubscriptionProperies();
-        const paymentPropertyList: SectionProperties[] = this.getPaymentProperies();
         return (
             <div className={AccountStyle.container}>
-                <AccountSection disabled={this.props.capturedLogs === ""} title={"User Details"} propertyList={userDetailPropertyList} />
-                <AccountSection title={"Subscription"} propertyList={subscriptionPropertyList} />
-                <AccountSection title={"Payment Information"} propertyList={paymentPropertyList} />
+                <AccountSection title={"User Details"} propertyList={this.getUserDetailProperies()} />
```
Comment (jperata) [inline src/components/MyAccount/Account.tsx]: typo getUserDetailProperties

File: src/components/MyAccount/Account.tsx [hunk 2]
```diff
@@ -125,14 +124,11 @@ export class Account extends React.Component<AccountProps, AccountState> {
...
+                <AccountSection title={"Subscription"} propertyList={this.getSubscriptionProperies()} />
```
Comment (jperata) [inline src/components/MyAccount/Account.tsx]: typo getSubscriptionProperties

File: src/components/MyAccount/Account.tsx [hunk 3]
```diff
@@ -125,14 +124,11 @@ export class Account extends React.Component<AccountProps, AccountState> {
...
+                <AccountSection title={"Payment Information"} propertyList={this.getPaymentProperies()} />
```
Comment (jperata) [inline src/components/MyAccount/Account.tsx]: typo getPaymentProperties

**Pull request merged**